### DISCUSSION TITLE
Alarms do not work in xDrip Sync Follower mode.

### DISCUSSION THREAD

**Discussion by cp2113** [NightscoutFoundation/xDrip#3718] [Bug reports]
I have a Xiaomi Redmi 6 type phone. It works perfectly in Carelink follower mode. If I switch to xDrip Sync Follower mode, it shows the value, but the alarms don't work. xDrip version: 2024.09.27 Nightly Bulid

**Comment by Navid200** [comment]
Please tell me exactly which alerts don't work.

Glucose level High and Low alerts?
Forecast Low Alert?
Persistent High Alert?
Fast rising/falling alert?
Sensor expiry alerts?
Calibration alerts?

**Reply by cp2113** [reply]
Glucose level high and low alerts. Sorry. Thank you. :-)

**Reply by Navid200** [reply]
Let me test this.  It may take a few hours.

**Reply by Navid200** [reply]
It works for me.

Can you post a screenshot of the classic status page?

**Reply by cp2113** [reply]
![462553878_1954530701627499_1448602824985166276_n](https://github.com/user-attachments/assets/346a4b9e-6281-4422-b5c1-c84309cc6ee4)

**Reply by cp2113** [reply]
In the meantime, I tried the versions up to the 2024.07.23 FCM problem, and the alarm works in the case of 07/23 variant1. It is interesting that not even with the 07/23 main version.
My phone is old, still running android 9, this is a possible problem.

**Reply by Navid200** [reply]
I don't know what you mean by FCM.
I don't understand what you mean by variant1.

Can you state clearly something like this?
Versions up to a work.
All versions after that fail.
Assuming that's what you want to say.

Then, it may be possible to figure out what change we have made could have caused this.

xDrip is compatible for Android 7 and newer.
It will be a very long time before we retire Android 9.  
So, this is not intentional.

**Comment by cp2113** [comment]
Hi! I'm sorry I didn't reply, but I was a little sick.
Maybe it's easier for me to answer your question that way:

You wrote: „I don't know what you mean by FCM.”
https://github.com/NightscoutFoundation/xDrip/issues/3582
You wrote: „I don't understand what you mean by variant1.”
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.07.23
 
![image](https://github.com/user-attachments/assets/eec9d1c7-e7ac-43b8-9495-4be141306d21)

If I install the application marked in yellow ([xDrip-plus-variant1-20240723-e6cb8d6.apk](https://github.com/NightscoutFoundation/xDrip/releases/download/2024.07.23/xDrip-plus-variant1-20240723-e6cb8d6.apk)), it works, but if I install the one before it ([xDrip-plus-20240723-e6cb8d6.apk](https://github.com/NightscoutFoundation/xDrip/releases/download/2024.07.23/xDrip-plus-20240723-e6cb8d6.apk) “main version”), it doesn't.
I appreciate your help and thank you.

**Reply by Navid200** [reply]
To use xDrip sync, you must use the current stable release (2024.09.17) or any after that.  
You cannot use older versions of xDrip.  Or else, xDrip sync will not work.  

Can you use the stable release?  You say that alarms don't work.
Can you disable `Use xDrip Cloud` on the xDrip sync settings page both on the master as well as the follower?
Does the alert work now?

**Comment by cp2113** [comment]
The sick person's phone runs xDrip-plus-20240927-23c06fc.apk (companion mode). I use xDrip-plus-variant1-20240723-e6cb8d6.apk for tracking (xDrip sync follower mode). It works. Thank you for your job.

**Reply by Navid200** [reply]
If you want to follow someone using xDrip sync, both master and follower phones must run the stable release, 2024.09.17, or later.
So, you should run the variant of 2024.09.17 or later on your phone (follower).

When you say it works, I take it the problem is resolved.  In that case, great.

**Comment by cp2113** [comment]
Thank you!
